lib/dry/view/exposures.rb

```diff
@@ -34,7 +34,10 @@ def bind(obj)
 
       def locals(input)
         tsort.each_with_object({}) { |name, memo|
-          memo[name] = self[name].(input, memo) if exposures.key?(name)
+          if exposures.key?(name)
+            value = self[name].(input, memo) || self[name].default_value
```

Comment:
Could the handling/returning of the default value be moved into `Exposure#call` instead of here? Feels to me like that better encapsulates the behaviour.

Reply:
Of course @timriley makes total sense